.github/workflows/template.flink-ci.yml

```diff
@@ -80,7 +80,7 @@ jobs:
       # timeout in minutes - this environment variable is required by uploading_watchdog.sh
       GHA_JOB_TIMEOUT: 240
     outputs:
-      stringified-workflow-name: ${{ steps.workflow-prep-step.outputs.stringified-workflow-name }}
+      stringified-workflow-name: ${{ steps.workflow-prep-step.outputs.stringified_value }}
     steps:
```

Comment:
@XComp, I kept this `outputs.stringified-workflow-name` as is, it is used in the later jobs. 

Initially I thought this could be confusing, but maybe all right to keep it as is, the `stringified_value` will be used only within the compile steps